webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js

```diff
@@ -100,6 +101,11 @@ const CVModuleStreamFilterContent = ({
     });
   }, [filterResults, filterId]);
 
+  const enableOriginalModuleStreams = (checked) => {
+    setOriginalModuleStreams(checked);
+    dispatch(editCVFilter(filterId, { original_module_streams: checked }));
```

Comment:
```suggestion
    setOriginalModueStreamsUpdating(true);
    setOriginalModuleStreams(checked);
    dispatch(editCVFilter(filterId, { original_module_streams: checked }, () => {setOriginalModueStreamsUpdating(false)}));
```

Something like this with a state to disable checkbox when updating would prevent multiple clicks..

Reply:
updated!.